doc/installation/system/inifile.rst

```diff
@@ -122,6 +122,28 @@ set ``PI_AUDIT_NO_SIGN = True``. If ``PI_AUDIT_NO_SIGN`` is set to ``True``
 audit entries will not be signed and also the signature of audit entries will not be
 verified. Audit entries will appears with *signature* *fail*.
 
+.. _monitoring_modules:
+
+Monitoring parameters
+---------------------
+
+``PI_MONITORING_MODULE`` lets you specify an alternative statistics monitoring module.
+The monitoring module takes care of writing values with timestamps to a store.
+This is used e.g. by the :ref:`eventcounter` and :ref:`taskmodule_simplestats`.
+
+The first available monitoring module is ``privacyidea.lib.monitoringmodults.sqlstats``.
+It accepts the following additional parameters:
+
+``PI_MONITORING_SQL_URI`` can hold an alternative SQL connect string. If not specified the
+normal ``SQL_DATABASE_URI`` is used.
```

Comment:
*``SQLALCHEMY_DATABASE_URI``

Reply:
done in d0d7b99c2734d452c7cd9c78c8037a45b29ba444

Reply:
:+1: